composer.json

```diff
@@ -145,6 +146,7 @@
 		],
 		"ci:php:cs-fixer": "php-cs-fixer fix --config .php-cs-fixer.php -v --dry-run --diff",
 		"ci:php:lint": "find .*.php *.php Classes Configuration Tests -name '*.php' -print0 | xargs -r -0 -n 1 -P 4 php -l",
+		"ci:php:mess": "phpmd Classes,Configuration,Tests text Build/phpmd.xml",
```

Comment:
PHPMD should only check `Classes`, not `Configuration` or `Tests`, as the code size in the tests does not really matter, and test cases tend to become big.